sdk/communication/azure-communication-chat/README.md

```diff
@@ -115,6 +114,11 @@ Perform Update operation on thread topic
 update_topic(topic, **kwargs)
 ```
 
+## Get Chat thread properties
+```python
+get_properties(**kwargs)
```

Comment:
The snippets will need a bit more context than this (and this applies to other existing snippets in the readme).
The goal is that a user should be able to copy+paste the code and use it right away.
So something more like:
```python
client = ChatClient(...)
thread = client.get_chat_thread_client(...)
thread.get_properties(...)
```

Given that this applies to multiple snippets throughout the readme - I'm happy if you want to open a github issue to address it in a subsequent PR.

Reply:
Will update the README and the samples in a separate PR